### PR TITLE
Updated Grafana Dashboard and screenshot

### PR DESCRIPTION
**Pull request by vegasbrianc**
Updated the Grafana dashboard and added it to the [Grafana Dashboards](https://grafana.com/dashboards/3434) directory. 

## Description
Created a new charts, logo, filters, and updated the layout of the Dashboard

## Motivation and Context
This update helps add some of the newer functionality available from Grafana to the dashboard
- [ ] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

## How Has This Been Tested?
This has been tested with a local instance running on my Mac. I have gone through the different scenarios of scaling, measuring execution times and it appears to be reporting correctly.  Please provide feedback as I will continue updating.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by vegasbrianc** [comment]
Forgot the sign...should be good now

**Comment by alexellis** [comment]
Thank you Captain - this looks great! Did the graphs/metrics change too? I think we should use the monochromatic logo from over at - https://github.com/openfaas/media/

**Comment by alexellis** [comment]
CC @developius @kenfdev

**Comment by alexellis** [comment]
Hi @vegasbrianc - the middle commit isn't signed-off. Please can you do this? Might be easiest to do a `git reset HEAD~3` and then commit again with `-s`.

**Comment by vegasbrianc** [comment]
Made a mess of this patch. I'm having little luck going back and signing previous commits

**Comment by alexellis** [comment]
I think with feedback from a few folks we'd like this to be an alternate/2nd option dashboard. Could you keep the original link too?

I've tried uploading the original dashboard but had issues on the Grafana UI. I re-created and exported via Grafana 4.x but got an error around "Cannot import Grafana 2.0 dashboard"

**Comment by alexellis** [comment]
Derek assign to: vegasbrianc

**Comment by vegasbrianc** [comment]
@alexellis I will add the original dashboard and commit it to Grafana as well.

File: README.md [hunk 1]
```diff
@@ -159,6 +159,8 @@ Anyone is invited to contribute to the project in-line with the [contribution gu
 
 Example of a Grafana dashboard linked to OpenFaaS showing auto-scaling live in action:
```
Comment (alexellis) [inline README.md]: I'm seeing " Example of a Grafana dashboard linked to OpenFaaS showing auto-scaling live in action:" duplicated. Please can you link to "Original" dashboard here too.

**Comment by vegasbrianc** [comment]
Hi @alexellis I added the original Dashboard to Grafana and converted it to Grafana 3.x.

**Comment by alexellis** [comment]
Merged, thanks

**Pull request merged**